arkane/statmech.py

```diff
@@ -401,31 +401,30 @@ def load(self, pdep=False):
         zpe_scale_factor = self.frequencyScaleFactor / 1.014
 
         logging.debug('    Reading energy...')
-        if E0_withZPE is None:
-            # The E0 that is read from the log file is without the ZPE and corresponds to E_elec
-            if E0 is None:
-                E0 = energyLog.loadEnergy(zpe_scale_factor)  # in J/mol
+        if e0 is None:
+            if e_electronic is None:
+                # The energy read from the log file is without the ZPE
+                e_electronic = energyLog.loadEnergy(zpe_scale_factor)  # in J/mol
             else:
-                E0 = E0 * constants.E_h * constants.Na  # Hartree/particle to J/mol
+                e_electronic = e_electronic * constants.E_h * constants.Na  # convert Hartree/particle into J/mol
```

Comment:
Can shorten to `e_electronic *= constants.E_h * constants.Na`

Reply:
done